monkey/infection_monkey/exploit/sshexec.py

```diff
@@ -188,12 +214,23 @@ def _exploit_host(self) -> ExploiterResultData:
                     self.exploit_result.error_message = f"SSH Skipping unknown os: {uname_os}"
 
                     if not uname_os:
+                        self.publish_propagation_event(
+                            target=self.host.ip_addr,
+                            propagation_success=False,
+                            error_message=self.exploit_result.error_message,
+                        )
+
```

Comment:
Why are we publishing a propagation event here? No attempt was made to propagate.

Reply:
Running a command on a system is a propagation event or am I wrong?